scenarios/centos-9/hci_ceph_backends.yml

```diff
@@ -24,26 +24,18 @@ cifmw_block_device_size: 20G
 
 cifmw_services_manila_enabled: true
 
-pre_tempest:
-  - name: Create manila resources
+pre_tests:
+  - name: 90 Create manila resources
```

Comment:
does the 90 have significance here ? (are pre_test ordered by task name?)

Reply:
@marios we have a `| sort` in the run_hooks role when we build the final list of hooks:
https://github.com/openstack-k8s-operators/ci-framework/blob/fe313ac79cd046a1b40a0b69c7ca8216aac12e46/roles/run_hook/tasks/main.yml#L68-L75

So using `90` as a prefix or anything like that would just ensure we have that hook "near the end of the list".

Reply:
Correct, for this job there is only one hook in `pre_tests` at this moment. I added just a remimder that, if we add another, we should check if there is a ordering needed between them.